### ISSUE TITLE
Ubuntu test failures

### ISSUE THREAD

**Issue by davidbrochart** [jupyterlab/jupyter-collaboration#416]
## Description

Compared to previous passing tests, there is this message in the [logs](https://github.com/jupyterlab/jupyter-collaboration/actions/runs/12258629145/job/34199042385):
```
[I 2024-12-10 14:56:40.178 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
```
But I'm not sure it's related.

**Comment by davidbrochart** [comment]
Yes, I meant apart from this error message. I cannot reproduce locally though.

**Comment by davidbrochart** [comment]
Fixed with JupyterLab v4.3.4.
